test/conformance/testing/uur/utils.h

```diff
@@ -42,6 +47,46 @@ inline std::string GetPlatformAndDeviceName(ur_device_handle_t device) {
     return GetPlatformName(GetPlatform()) + "__" + GetDeviceName(device);
 }
 
+inline std::pair<uint32_t, bool>
+urContextGetReferenceCount(ur_context_handle_t context) {
```

Comment:
Since these are in the `uur` namespace the `ur` prefix seems unnecessary and potentially misleading as these are not part of the spec.